uiowa/uiowa
# OnIowa Event Map

Resolves #7052 

# How to test
- `ddev composer install && ddev blt ds --site=oniowa.uiowa.edu && ddev drush @oniowa.local uli`
- Check an existing event from https://oniowa.uiowa.ddev.site/events, confirm that the sidebar now renders a map view from OpenStreetMap instead of a link to Google Maps.
- Create an event, confirm that the geofield is still set to take in lat and long values
- Check an event with no lat/long values to confirm no map appears
- Test other sites using the event feature (admissions, iowasummerwritingfestival) also work as expected.

Currently the map is set to not appear if no lat/long, and to lazy load if there is. Open to any feedback about those choices, the Zoom level, and anything else.

- `ddev blt ds --site=pharmacy.uiowa.edu && ddev blt ds --site=admissions.uiowa.edu` and check that the https://pharmacy.uiowa.ddev.site/pharmd/dual_cert/palliative/certificate_graduates and https://admissions.uiowa.ddev.site/counselors still render the map and that the map assets load from the module instead of /libraries
